kruize-clowdapp.yaml

```diff
@@ -79,10 +99,7 @@ objects:
       - name: create-kruize-partitions
         schedule: "0 0 25 * *" # Run on 25th of every month at midnight
         podSpec:
-          name: kruizecronjob
           image: ${KRUIZE_IMAGE}:${KRUIZE_IMAGE_TAG}
-          imagePullPolicy: Always
-          restartPolicy: OnFailure
```

Comment:
Any reason we are removing this?

Reply:
I have posted a screenshot above along with a comment. Additionally these params looked irrelevant for a job.

Reply:
Hope you have tested the changes in EE after removing above attributes. Rest of the PR looks good to me so approving the PR.

Reply:
Yup changes tested on EE, thanks.